### PR TITLE
updating mbezhanov/faker-provider-collection for using FakerPHP/Faker

### PR DESCRIPTION
**Pull request by robinflyhigh**
…P/Faker

### CODE REVIEW

**Comment by robinflyhigh** [comment]
@ghermans @jitendra-webkul

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ This module allows you to generate fake data that can be populated into the data
 ### 3. Installation:
 
 ```sh
-composer require bagisto/laravel-datafaker
+composer require robinflyhigh/bagisto-laravel-data-faker
```
Comment (ghermans) [inline README.md:20]: This should not be changed, please revert this back to `bagisto/laravel-datafaker`.

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "bagisto/laravel-datafaker",
+    "name": "robinflyhigh/bagisto-laravel-data-faker",
```
Comment (ghermans) [inline composer.json:2]: This should not be changed, please revert this back to
```json
"name": "bagisto/laravel-datafaker",
```

**Review by ghermans — CHANGES_REQUESTED** [review]

**Review by ghermans — COMMENTED** [review]
@robinflyhigh please verify the following

**Comment by palmatesolutions** [comment]
Ignore my pull request. Just update the package.json file

from 
        "mbezhanov/faker-provider-collection": "^1.2"
to
        "mbezhanov/faker-provider-collection": "^2.0"

**Pull request closed without merging**